src/components/Dialog/Newconnections.vue

```diff
@@ -74,92 +77,91 @@
-import Avatar from "primevue/avatar"
-import Dropdown from "primevue/dropdown"
-import InputText from "primevue/inputtext"
-import Button from "primevue/button"
-import Panel from "primevue/panel"
-import Projects from "@/store/Modules/Projects"
-import { getModule } from "vuex-module-decorators"
-const appProjects = getModule(Projects)
-console.log(appProjects.projectList)
+  import Dialog from "primevue/dialog"
+  import Avatar from "primevue/avatar"
+  import Dropdown from "primevue/dropdown"
+  import InputText from "primevue/inputtext"
+  import Button from "primevue/button"
+  import Panel from "primevue/panel"
+  import Projects from "@/store/Modules/Projects"
+  import { getModule } from "vuex-module-decorators"
+  const appProjects = getModule(Projects)
 
-export default {
-  name: "NewConnections",
-  components: {
-    Dialog,
-    Avatar,
-    Panel,
-    InputText,
-    Button,
-    Dropdown,
-  },
-  props: {
-    connectiondialog: { type: Boolean, default: false },
-  },
-  data() {
-    return {
-      type: "Connection",
-      name: "",
-      expanded: "",
-      description: "",
-      icon: "",
-      display: true,
-      selected: null,
-    }
-  },
-  computed: {
-    projectList() {
-      return appProjects.projectList
+  export default {
+    name: "NewConnections",
+    components: {
+      Dialog,
+      Avatar,
+      Panel,
+      InputText,
+      Button,
+      Dropdown,
     },
-  },
-  emits: ["close"],
-  methods: {
-    conncetioncloseDialog() {
-      this.$emit("close")
+    props: {
+      connectiondialog: { type: Boolean, default: false },
     },
-    handleconnectionstore() {
-      const data = {
-        name: this.selected,
-        list: {
-          name: this.name,
-          icon: this.icon,
-          description: this.des,
-          type: "connection",
-        },
+    data() {
+      return {
+        type: "Connection",
+        name: "",
+        expanded: "",
+        description: "",
+        icon: "",
+        display: true,
+        selected: null,
       }
-      appProjects.addNewConnection(data)
-      this.$emit("close")
     },
-  },
-}
+    computed: {
+      projectList() {
+        return appProjects.projectList
+      },
+    },
+    emits: ["close"],
+    methods: {
+      conncetioncloseDialog() {
+        this.$emit("close")
+      },
+      handleconnectionstore() {
+        const data = {
+          name: this.selected,
+          list: {
+            name: this.name,
+            icon: this.icon,
+            description: this.des,
+            type: "connection",
+          },
+        }
+        appProjects.addNewConnection(data)
+        this.$emit("close")
+      },
+    },
+  }
 </script>
-<style  lang="scss">
-input {
-  width: 80%;
-}
-.field {
-  display: grid;
-}
-.connection-dialog {
-  height: 100vh;
-  width: 40vw;
-  .p-dropdown {
+<style lang="scss">
+  input {
     width: 80%;
   }
-  .p-dialog-content {
-    height: 100%;
+  .field {
```

Comment:
Do not modify Prime Vue Class Directly.

Reply:
this issue have been resolved in ticket #149